css/license.scss

```diff
@@ -0,0 +1,50 @@
+@import "global";
+
+.license {
```

Comment:
This one is a good example of BEM usage!
`license` is a block and `__single`, `__price` etc are its elements. And if price could be discounted you could add modifier '_discount` to it.